src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.targets

```diff
@@ -177,8 +223,12 @@
     BeforeTargets="BuiltProjectOutputGroup"
     Condition="'$(ResolvedRazorCompileToolset)'=='RazorSdk' and '$(RazorCompileOnBuild)'=='true'">
 
+    <PropertyGroup>
+      <RazorOutputPath Condition="'$(RazorOutputPath)'==''">$([MSBuild]::EnsureTrailingSlash('$(OutDir)'))</RazorOutputPath>
+    </PropertyGroup>
```

Comment:
Due to evaluation order we have to set the default value of this in the two places we use it. I didn't feel like adding another target was clearer.